chia/wallet/vc_wallet/vc_wallet.py

```diff
@@ -105,6 +105,10 @@ async def coin_added(self, coin: Coin, height: uint32, peer: WSChiaConnection) -
             return  # pragma: no cover
         vc = VerifiedCredential.get_next_from_coin_spend(cs)
         vc_record: VCRecord = VCRecord(vc, height)
+        # Send notification
+        self.wallet_state_manager.state_changed(
+            "vc_received", self.id(), dict(launcher_id=vc_record.vc.launcher_id.hex())
```

Comment:
Naming suggestion to match the loose convention we have for these notifications. e.g. `nft_coin_added` 
```suggestion
            "vc_coin_added", self.id(), dict(launcher_id=vc_record.vc.launcher_id.hex())
```